pkg/cmd/server/origin/asset_apiserver_adapter.go

```diff
@@ -25,6 +26,13 @@ func NewAssetServerConfigFromMasterConfig(masterConfigOptions configapi.MasterCo
 		return nil, err
 	}
 
+	backend, policy, err := kubernetes.GetAuditConfig(masterConfigOptions.AuditConfig)
```

Comment:
Find the call site of `NewAssetServerConfigFromMasterConfig` and set the backend and policy there.